src/main/java/edu/cornell/kfs/pdp/batch/service/impl/PayeeACHAccountExtractServiceImpl.java

```diff
@@ -338,7 +339,18 @@ protected void cleanPayeeACHAccountExtractDetail(PayeeACHAccountExtractDetail de
             if (!StringUtils.isNumeric(detail.getBankAccountNumber())) {
                 LOG.error(logMessageStarter + " is not numeric after cleaning");
             }
+
+            detail.setBankName(extractBankName(detail));
+        }
+    }
+
+    private String extractBankName(PayeeACHAccountExtractDetail detail) {
+        String bankName = ObjectUtils.isNull(detail) ? StringUtils.EMPTY : detail.getBankName();
+        if (StringUtils.length(bankName) > 40) {
+            LOG.info("cleanPayeeACHAccountExtractDetail truncating bank name to 40 characters");

```

Comment:
This log statement should be updated to reference the "extractBankName" method that it's actually inside of.